propTypes/campaign.js

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types'
+import imagePropTypes from './image'
+
+export default {
```

Comment:
If we don't have a Campaign type, what do these propTypes belong to?

Reply:
https://github.com/futurice/masifunde-fe/pull/207/files#diff-33691826b1a039388d8652a492cb1382R171